### PR TITLE
Introduction Edits

### PR DESCRIPTION
**Pull request by localghost443**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -101,71 +117,69 @@ module.exports = {
...
+// modules/@apostrophecms/image-widget/index.js
 module.exports = {
   options: {
-    className: 'a3-widget-image'
+    className: 'home-image'
   }
 };
-```
-
-Now you can add CSS so images don't run off the page. Add this to `./src/index.scss`:
```
Comment (localghost443) [inline docs/README.md:163]: I removed this for now because I wasn't convinced that we should tell them to put their scss here. Open to thoughts on this in terms of setting them up full with a robust opinion here.

File: docs/README.md [hunk 2]
```diff
@@ -1,52 +1,68 @@
...
 # Introduction
 
-This document is for Apostrophe 2.x developers who wish to try out Apostrophe 3.x and give feedback. We look forward to your input!
+This document assumes a level of familiarity with Apostrophe 2. We suggest building your production projects with the latest stable release of [Apostrophe 2](https://github.com/apostrophecms/apostrophe). This repository reflects our current progress towards a stable 3.0 release, which is scheduled in Q1 of 2021. We're eager to share our progress so far and grateful for any feedback you might have.
 
-## Getting Started
+### New in Apostrophe 3
 
-Grab the boilerplate project and give your project a name of its own. We're assuming `myproject` here. Replace that with your own project name wherever you see it. Use a short name with only letters, digits and dashes.
+- New editing experience
```
Comment (boutell) [inline docs/README.md:11]: I will accept and then do another PR filling in the gaps here.

File: docs/README.md [hunk 3]
```diff
@@ -1,52 +1,68 @@
...
-git clone https://github.com/apostrophecms/a3-boilerplate myproject
-cd myproject
-npm install
-node app @apostrophecms/user:add admin
-npm run dev
-```
+### Coming Soon in Future Releases
 
-> Be sure to give the admin user a password when prompted.
+- Permissions (all admins)
```
Comment (boutell) [inline docs/README.md:19]: I'll do this in my PR.

File: docs/README.md [hunk 4]
```diff
@@ -101,71 +117,68 @@ module.exports = {
...
 ```
 mkdir modules/@apostrophecms/image-widget
 ```
 
-Then you can configure it:
+Then, you can configure it by creating an `index.js` file in that directory.
 
-```
-// in modules/@apostrophecms/image-widget/index.js
+```jsx
```
Comment (boutell) [inline docs/README.md:176]: It's just js.

File: docs/README.md (hunk 1 above)
Comment (boutell) [reply docs/README.md:163]: Please don't remove — instead I promise to change it in accordance with what Bea and I work out when we work it out.

File: docs/README.md [hunk 5]
```diff
@@ -101,71 +117,68 @@ module.exports = {
...
-**"What's changed from A2 in this code?"**
-
-* Project-level modules now reside in `./modules`, not `./lib/modules`.
+### Modules
 
-* Apostrophe's core modules like `@apostrophecms/home-page` are namespaced now, just like newer npm modules. This is true even for modules that ship inside a "bundle" such as the `apostrophe` npm module.
+- Project-level modules have been moved from `./lib/modules` to  `./modules`
+- Apostrophe's core modules like `@apostrophecms/home-page` are namespaced now, just like newer npm modules. This is true even for modules that ship inside a "bundle" such as the `apostrophe` npm module.
 
-* Even the home page has its own module, `@apostrophecms/home-page`. We're just configuring it here, so we don't have to use `extend`. But we will when we add a new page type to the site.
```
Comment (boutell) [inline docs/README.md:110]: Why was this removed?

File: docs/README.md [hunk 6]
```diff
@@ -101,71 +117,68 @@ module.exports = {
...
+### Schemas
 
-* Ordinary module options like `label` now go in an `options` property, not at the top level.
+- Ordinary module options like `label` now reside in an `options` property.
+- Every area must be configured as part of the schema for a page-type or piece-type.
+- Fields are added to the module with the new `fields` syntax. New fields are added via the `add` property.
+- You can also `remove` fields by passing an array for `remove`.
+- For convenience, fields are now configured as an object, rather than an array.
 
-* Every area must be configured as part of the schema for a page type or piece type.
```
Comment (boutell) [inline docs/README.md:114]: This is a super major A3 change that people need to see.

File: docs/README.md [hunk 7]
```diff
@@ -101,71 +117,68 @@ module.exports = {
...
+### Templates
 
-* Fields are added to the module with the new `fields` feature. New fields go in the `add` property. You can also `remove` fields by passing an array for `remove`. For convenience, fields are now configured as an object, rather than an array.
```
Comment (boutell) [inline docs/README.md:116]: Why was this removed?

File: docs/README.md [hunk 8]
```diff
@@ -101,71 +117,68 @@ module.exports = {
...
 
-**"What's changed from A2 in this code?"**
-
-1. Areas are added to the template with the new `area` nunjucks tag. There is no `apos.area` helper function anymore.
-
-2. You don't configure the area here. You do that in the `index.js` file for the page type or piece type. In the template you just pass the page and the name of the area.
+```
 
-### Adding more standard widgets
+### Widgets
```
Comment (boutell) [inline docs/README.md:151]: How about just "Standard Widgets"

As distinct from custom which will come later

File: docs/README.md [hunk 9]
```diff
@@ -101,71 +117,68 @@ module.exports = {
...
 
 Try adding more sub-properties to `widgets` in `index.js`:
 
-```javascript
-'@apostrophecms/video': {},
-'@apostrophecms/raw-html': {},
-'@apostrophecms/image': {}
+```jsx
```
Comment (boutell) [inline docs/README.md:155]: Just js if not javascript

Comment (boutell) [reply docs/README.md:155]: jsx is a react thing

File: docs/README.md [hunk 10]
```diff
@@ -101,71 +117,68 @@ module.exports = {
...
+// modules/@apostrophecms/home-page/index.js
+
+...
+widgets: {
+	'@apostrophecms/video': {},
```
Comment (boutell) [inline docs/README.md:160]: No tabs! Two spaces.

File: docs/README.md [hunk 11]
```diff
@@ -101,71 +117,68 @@ module.exports = {
...
+	'@apostrophecms/raw-html': {},
+	'@apostrophecms/image': {}
+...
 ```
 
-You will also need to configure a CSS class for the image widget, because A3 does not impose strong opinions on your front end code. First make a folder for project-level configuration of the image-widget module:
+**Configuring a Widget**
```
Comment (boutell) [inline docs/README.md:166]: Use a heading.

File: docs/README.md [hunk 12]
```diff
@@ -101,71 +117,68 @@ module.exports = {
...
+
+Apostrophe 3 does not impose any front-end opinions regarding widgets, and thus its necessary to configure them properly with CSS classes for styling. In this example, we'll configure our image-widget to have a class. Start by creating a directory for project-level configuration. Using your terminal:
```
Comment (boutell) [inline docs/README.md:168]: it's

File: docs/README.md [hunk 13]
```diff
@@ -101,71 +117,68 @@ module.exports = {
...
+Then, you can configure it by creating an `index.js` file in that directory.
 
-```
-// in modules/@apostrophecms/image-widget/index.js
+```jsx
+// modules/@apostrophecms/image-widget/index.js
 module.exports = {
   options: {
-    className: 'a3-widget-image'
+    className: 'home-image'
```
Comment (boutell) [inline docs/README.md:180]: This is not specific to the home page so definitely not `home-`. The a3 prefix is a nice convention to separate it from custom widgets in the project, but it's not essential.

File: docs/README.md [hunk 14]
```diff
@@ -1,52 +1,68 @@
...
-title: 'A3 Docs Home'
+title: 'Apostrophe 3 Documentation'
 ---
 
-> Apostrophe 3 is currently in its alpha 1 release. You should build production projects with [Apostrophe 2](https://docs.apostrophecms.org).
-
 # Introduction
 
-This document is for Apostrophe 2.x developers who wish to try out Apostrophe 3.x and give feedback. We look forward to your input!
+This document assumes a level of familiarity with Apostrophe 2. We suggest building your production projects with the latest stable release of [Apostrophe 2](https://github.com/apostrophecms/apostrophe). This repository reflects our current progress towards a stable 3.0 release, which is scheduled in Q1 of 2021. We're eager to share our progress so far and grateful for any feedback you might have.
```
Comment (boutell) [inline docs/README.md:7]: 👍

File: docs/README.md [hunk 15]
```diff
@@ -1,52 +1,68 @@
...
-## Making your project your own
+::: tip Note: 
+When you make code changes the boilerplate project will automatically restart and refresh the browser. If you get a "port in use" error, press control-C and start `npm run dev` again. We're tracking down how to reliably reproduce this issue.* 
+:::
 
-First **edit `app.js` and change `shortName` to `myproject`** (replace with the folder name of your own project).
+## Getting Started
 
-Then, to push your project to your own github, cut it loose from ours:
+Clone the Apostrophe 3 Boilerplate project give your project a name of its own. Legal characters consist of letters, digits and dashes. We're assuming **myproject** here. Be sure to give the admin a user and password when prompted.
```
Comment (boutell) [inline docs/README.md:44]: Two sentences ran together here.

I see you want to skip talking about how to get this project into their own repo. I think it's nice, but it's not essential for now.

File: docs/README.md (hunk 5 above)
Comment (boutell) [reply docs/README.md:110]: It's a key difference from A2 pages.

File: docs/README.md [hunk 16]
```diff
@@ -1,52 +1,68 @@
...
-## Editable content areas in Apostrophe 3
+::: tip Note: 
+Currently, all edits made contextually are automatically saved. We're introducing an Edit Mode in our next released, which will offer a direct "save" option and remove automatic saving.
+:::
 
-### `index.js` for the home page
+## Developing in Apostrophe 3
 
-You'll find this in `modules/@apostrophecms/home-page/index.js`:
+Apostrophe 3 introduces several changes to module architecture and schemas. Lets start by taking a look at our home page and discuss the major differences.
```
Comment (boutell) [inline docs/README.md:62]: Let's

**Review by boutell — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 13 above)
Comment (localghost443) [reply docs/README.md:180]: Point taken, but we shouldn't recommend using Apostrophe as a reference when making classes....how about 
`full-width-image`, since that's what max-width 100% does.

File: docs/README.md (hunk 1 above)
Comment (localghost443) [reply docs/README.md:163]: Ok

File: docs/README.md (hunk 5 above)
Comment (localghost443) [reply docs/README.md:110]: roger

File: docs/README.md (hunk 6 above)
Comment (localghost443) [reply docs/README.md:114]: This wasn't removed, it was moved to the shema section breakdown

File: docs/README.md (hunk 7 above)
Comment (localghost443) [reply docs/README.md:116]: was not removed

File: docs/README.md (hunk 10 above)
Comment (localghost443) [reply docs/README.md:160]: LOL

**Review by boutell — APPROVED** [review]
